guides/release-process.md

```diff
@@ -70,16 +70,12 @@ You can build the Cypress binary locally by running `yarn binary-build`. You can
 
 ### Before Publishing a New Version
```

Comment:
In the Pre-reqs section, we can remove the need for the `cypress-bot` credentials.

Reply:
`do:comment` requires those credentials: https://github.com/cypress-io/release-automations/tree/master/packages/issues-in-release#comment

Reply:
Ye. Added more info around it in e8f523299a